README.md

```diff
@@ -90,6 +90,58 @@ Please note that if you use a kubernetes cluster created on DigitalOcean, there
 will be a cloud controller manager running in the cluster already, so you local
 one will compete for API access with it.
 
+### Run Locally (optional features)
+
+#### Add Public Access Firewall
+
+If you want to add an additional firewall, that allows public access to your
+cluster, you can run a command like this:
+
+```bash
+cd cloud-controller-manager/cmd/digitalocean-cloud-controller-manager
+FAKE_REGION=fra1 DO_ACCESS_TOKEN=your_access_token            \
+PUBLIC_ACCESS_FIREWALL_NAME=firewall_name                     \
+PUBLIC_ACCESS_FIREWALL_TAGS=k8s,k8s:<cluster-uuid>,k8s:worker \
+go run main.go                                                \
+  --kubeconfig <path to your kubeconfig file>                                                      
```

Comment:
nit: while I recognize this is the established way of writing commands, I think it's actually harder to read to try to fit everything on a single command line call using line continuations like this, but as long as we're doing it we should also add one on this line. 

I will submit a PR after yours is merged to propose an alternate way of writing this that is easier on the eyes.